### DISCUSSION TITLE
How do i update the mods?

### DISCUSSION THREAD

**Discussion by Isuckwithusernames** [winstxnhdw/lc-hax#140] [Q&A] [answered]
I downloaded this 2 days ago and some of the mods dont work (ex: /mob, /mask, /hate, and some of the newer commands) also ive seen that some of the triggermods dont work (unless im just using it incorrectly).

**Comment by winstxnhdw** [comment, marked as answer]
See #80

**Reply by Isuckwithusernames** [reply]
ill take a look at it, thanks

**Reply by Isuckwithusernames** [reply]
I tried the `git pull` command in GIT Bash (which is where im assuming im supposed to do it) but i get the error: `fatal: not a git repository (or any of the parent directories): .git`

im very new to "coding" so sorry if i dont understand lol

**Reply by winstxnhdw** [reply]
You should be doing it within the repository that you've cloned.

**Reply by Isuckwithusernames** [reply]
where would i find that?

**Reply by winstxnhdw** [reply]
Where did you clone the repository?

**Reply by Isuckwithusernames** [reply]
i have no idea, when i was installing it i just copied and pasted everything from the main page

**Reply by Isuckwithusernames** [reply]
when you say repository is that just the folder lc-hax?

**Reply by winstxnhdw** [reply]
Yes

**Reply by Isuckwithusernames** [reply]
ohh i see, i know where that is i just dont know where im supposed to type git pull into

**Reply by winstxnhdw** [reply]
Within the folder

**Reply by Isuckwithusernames** [reply]
sorry im very confused lmao within which folder? i see .config .github bin lc-hax submodules 
.editorconfig .gitignore .gitmodules launch.bat README.md and requirements.bat

**Reply by winstxnhdw** [reply]
You are already in the folder if you can see those. Just do `git pull` here.

**Reply by Isuckwithusernames** [reply]
where do i type it though? on the other post it said i need to type it into terminal

**Reply by winstxnhdw** [reply]
Yeap, you open the terminal inside the folder.

**Reply by Isuckwithusernames** [reply]
which folder has it? i dont see any folder that specifically says terminal. could you give me the name of it

**Reply by Isuckwithusernames** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/102493377/9741bd25-4fc4-48fe-b7e2-1f359de79f2e)

**Reply by winstxnhdw** [reply]
It's not a file/folder. You have to open git bash here.

**Reply by Isuckwithusernames** [reply]
im not following, where do i launch git bash from? can i just launch it from the windows search bar?what exactly do i need to type?

**Reply by winstxnhdw** [reply]
https://stackoverflow.com/questions/72100187/how-to-open-git-bash-from-specific-folder-in-windows-11

**Reply by Isuckwithusernames** [reply]
oh now i understand, i didnt know i could launch terminal from a specific folder. it is working now, thanks so much for ur help
